index.html

```diff
@@ -1,85 +1,53 @@
+ <link rel="stylesheet" href="style.css">
+ <title>Portfolio</title>
 </head>
 <body>
 
-    <div class= "container">
-
-        <nav class= "flex-nav"> 
-
-            <label class=" logo"> Vincent A. Obunga/ SOFTWARE DEVELOPER </label>
-
-            <ul>
-                <li> <a class="active" href="#">HOME </a></li>
-                <li> <a href="#">ABOUT ME </a></li>
-                <li> <a href="#">RESUME</a> </li>
-                <li> <a href="#">PROJECTS</a></li>
-                <li> <a href="#">CONTACT</a></li>
-            </ul>     
-        </nav>
-
-        <div class= "content">
-
-            <div class="box box1 "> 
-
-                <img src="C:\Users\PCH.TL1\Desktop\Portfolio\pics\passport.jpeg" alt="passport" 
-                style="width:200px;height:200px;">
-                
-                <h3>Vincent Abuya Obunga </h3>
-                <h5>Software Developer </h5>
-                <h5>vabuyia@gmail.com</h5>
-            </div>
-
-            <div class="box box2 ">
-
-                <div class="about">
-                    
-                    <h4>ABOUT ME</h4>
-                    <p>I was born in 1983. I live in Basra and work as a Software Developer/Architect. <br> 
-                        I’ve been a professional developer for more than 2 years. <br> 
-                        I’ve mostly worked professionally with HTML, CSS, JavaScript, Ruby, Perl and PHP,<br> 
-                        but also little bit with Java and Python. C, C++. 
-                    </p>                         
-                </div>
-
-                <div class="social-icon">
-
-                    <h4> CONTACT </h4>
-
-                    <a href="https://twitter.com/abuyiag" target="_blank"> 
-                        <ion-icon name="logo-twitter"></ion-icon>  
-                    </a>
-
-                    <a href="linkedin.com/in/vincent-abuya-a1940555" target="_blank"> 
-                        <ion-icon name="logo-linkedin"></ion-icon> 
-                    </a>   
-
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a> 
-                    <a href="https://github.com/vabuyia" target="_blank"> 
-                        <ion-icon name="logo-github"></ion-icon>
-                    </a>
-                                        
-                </div> 
-
-            </div>
-
-        </div>
-
-                          
-    </div>
-
-        
-    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
-    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+<header>
+  <img src="wccl.png" alt=" " class="logo ">
+
+  <nav> 
+    <ul class= "nav-area">
+      <li> <a href="#" >Portfolio</a></li>
+      <li> <a href="#" >About</a></li>
+      <li> <a href="#" >Contact</a></li>
+    </ul>
+  </nav>
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/133863210-8c804435-5f29-4776-8cde-ac68a3d17074.png)

- [ ] As you can see in the picture of the original design. For the mobile view, instead of the links only the menu icon is showing. Kindly fix the issue.